CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to `src-cli` are documented in this file.
 ## Unreleased
 
 ### Added
+- `src code-intel` is a new alias for `src lsif`. [#745](https://github.com/sourcegraph/src-cli/pull/745)
```

Comment:
<3

Reply:
Oops, that is linking the wrong PR number.